reactor-netty-core/src/test/java/reactor/netty/ConnectionTest.java

```diff
@@ -72,12 +67,11 @@ public void addByteDecoderWhenNoLeft() {
 		           .addHandlerFirst("decoder$extract",
 				           NettyPipeline.inboundHandler(ADD_EXTRACTOR));
 
-		assertEquals(channel.pipeline()
-		                    .names(),
-				Arrays.asList("decoder$extract",
+		assertThat(Arrays.asList("decoder$extract",
```

Comment:
not terribly important but it is better to place the value under test in the `assertThat`, so that adding assertions will work on the actual value, not the expected value. consider also simply using `containsExactly`, which will list extra/missing items if the assertion fails:
```java
assertThat(channel.pipeline().names()).containsExactly("decoder$extract", "decoder", ...);
```

Reply:
the above comment applies to most (all?) tests in this class